modules/python/src2/cv2.cpp

```diff
@@ -1627,6 +1647,27 @@ template<typename _Tp> static inline PyObject* pyopencv_from_generic_vec(const s
     return seq;
 }
 
+template<>
+PyObject* pyopencv_from_generic_vec<bool>(const std::vector<bool>& value)
+{
+    int i, n = (int)value.size();
+    PyObject* seq = PyList_New(n);
+    for( i = 0; i < n; i++ )
+    {
+        bool elem = value[i];
+        PyObject* item = pyopencv_from(elem);
```

Comment:
It makes sense to reuse generic implementation above with minor modifications:

```
_Tp elem = value[i];
PyObject* item = pyopencv_from(elem);
```

Reply:
Fixed

Reply:
Not fixed, sorry. Because not all structures in `opencv_contrib` has a default ctor, so it cause warnings.